server/pyproject.toml

```diff
@@ -38,13 +38,14 @@ boto3 = "^1.28.34"
 urllib3 = "<=1.26.18"
 hqq = { version = "^0.1.2", optional = true }
 stanford-stk = { version = "^0.7.0", markers = "sys_platform == 'linux'" }
+aqlm = { version = "^1.0.0"}
```

Comment:
Proper backprop was added in `aqlm==1.1.0` and bf16 support was added in `aqlm==1.1.2`.